org.spyder_ide.spyder.yaml

```diff
@@ -0,0 +1,149 @@
+app-id: org.spyder_ide.spyder
+runtime: org.kde.Sdk
+runtime-version: 5.15-22.08
+sdk: org.kde.Sdk
+command: spyder-wrapper
+rename-desktop-file: spyder.desktop
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: 5.15-22.08
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --filesystem=home
```

Comment:
Why do you need full home? Does it support filechooser portals?

Reply:
It does support portal. But when trying to run Python file, the directory becomes /run/user/ which then would fail to run. Not sure how to fix it.

Reply:
Yea I don't think that's going to work without direct filesystem access